.github/workflows/ruff.yml

```diff
@@ -16,4 +16,6 @@ jobs:
       - name: Install dependencies
         run: pip install -r requirements.txt
       - name: Ruff Check
-        run: ruff check .
+        run: ruff check
+      - name: Ruff Format
+        run: ruff format
```

Comment:
Do not change the code via github action. Try: `ruff format --check` instead?

Reply:
My bad. Copied from another project I work on. The action doesn't commit or anything, so I was just expecting this to fail do to a non-0 exit code.

Changed.